src/main/java/br/codenation/hospital/manager/configuration/Instantiation.java

```diff
@@ -1,42 +1,109 @@
 
   @Override
   public void run(String... args) throws Exception {
 
     hospitalRepository.deleteAll();
+    patientRepository.deleteAll();
 
-    final Hospital samaritanHospital =
+     Hospital samaritanHospital =
         new Hospital(
             "Hospital Samaritano",
             "R. Conselheiro Brotero, 1486",
-            1000L,
-            600L,
+            10L,
+            10L,
             -23.5392135,
             -46.66193084);
 
-    final Hospital morumbiHospital =
+     Hospital morumbiHospital =
         new Hospital(
             "Hospital São Luiz Unidade Morumbi",
             "Rua Engenheiro Oscar Americano, 840 - Jardim Guedala, São Paulo - SP, 05605-050",
-            10057L,
-            785L,
+            5L,
+            5L,
             -23.59067495,
             -46.67325182);
 
+     Patient paciente1 =
+            new Patient("Pedro Silva",
+                    "0123456789",
+                    LocalDate.of(1992,2,14),
+                    "male",
+                    null,
+                    31.7654,
+                    52.3376);
+
+     Patient paciente2 =
+            new Patient("Lucas Santos",
+                    "9876543210",
+                    LocalDate.of(1994,5,23),
+                    "male",
+                    null,
+                    23.5505,
+                    46.6333);
+
+     Patient paciente3 =
+              new Patient("Regina Souza",
+                      "555555555",
+                      LocalDate.of(1978,5,21),
+                      "female",
+                      null,
+                      32.0395,
+                      52.1014);
+
+      SupplyItem seringa =
+              new SupplyItem("1","Seringa",Long.valueOf(5000), SupplyType.PRODUCT);
+      SupplyItem gesso =
+              new SupplyItem("2","Gesso",Long.valueOf(250), SupplyType.PRODUCT);
+      SupplyItem gaze =
+              new SupplyItem("3","Gaze",Long.valueOf(45000), SupplyType.PRODUCT);
+      SupplyItem sangueAB =
+              new SupplyItem("4","Sangue AB",Long.valueOf(6), SupplyType.BLOOD_BANK);
+      SupplyItem sangueOPos =
+              new SupplyItem("5","Sangue O Positivo",Long.valueOf(25), SupplyType.BLOOD_BANK);
+
+    patientRepository.saveAll(Arrays.asList(paciente1,paciente2,paciente3));
     hospitalRepository.saveAll(Arrays.asList(samaritanHospital, morumbiHospital));
+
+    hospitalService.insertProduct(samaritanHospital.getId(),seringa);
+    hospitalService.insertProduct(samaritanHospital.getId(),seringa);
+    hospitalService.insertProduct(samaritanHospital.getId(),seringa);
+    hospitalService.insertProduct(samaritanHospital.getId(),seringa);
+
+    hospitalService.insertProduct(morumbiHospital.getId(),sangueAB);
+
+    hospitalService.checkinPatient(patientService.loadPatient(paciente1.getId()),samaritanHospital.getId());
+
+     // Banco Inicializado com 2 Hospitais, 3 pacientes e 5 Itens, sendo 3 produtos e 2 bancos de sangue//
```

Comment:
What isn't necessary? 
I used the loadPatient method to obtain the patient object with its respective DB ID, since it is just obtainded after the insertion.

Reply:
The comments. They are not necessary since the code is extremely straight forward